appshared/src/test/java/io/reark/rxgithubapp/shared/utils/NullNetworkInstrumentationTest.java

```diff
@@ -45,10 +43,7 @@ public void setUp() throws Exception {
     @Test
     public void testDecorateNetwork_DoesNotChangeTheHttpClient() {
         OkHttpClient okHttpClient = mock(OkHttpClient.class);
-
         instrumentation.decorateNetwork(okHttpClient);
-
```

Comment:
I just meant that leave the two line breaks that you removed from inside the test method, they are there on purpose to separate the three lines.